tools/vms-generator/utils/utils.go

```diff
@@ -122,6 +123,22 @@ func getBaseVMI(name string) *v1.VirtualMachineInstance {
 func initFedora(spec *v1.VirtualMachineInstanceSpec) *v1.VirtualMachineInstanceSpec {
 	addContainerDisk(spec, fmt.Sprintf("%s/%s:%s", DockerPrefix, imageFedora, DockerTag), busVirtio)
 	addRNG(spec) // without RNG, newer fedora images may hang waiting for entropy sources
+
+	return spec
+}
+
+func setDefaultNetworkAndInterface(spec *v1.VirtualMachineInstanceSpec, bindingMethod v1.InterfaceBindingMethod, networkSource v1.NetworkSource) *v1.VirtualMachineInstanceSpec {
+	spec.Domain.Devices.Interfaces = []v1.Interface{
+		v1.Interface{
+			Name:                   defaultName,
```

Comment:
I believe you should have renamed it here too...

Reply:
oops. fixed